src/ParkChecklist/ParkChecklist.js

```diff
@@ -157,57 +147,17 @@ const ParkChecklist = ({ apiLink }) => {
           </div>
           <div className="sorting-options">
             <div className="sort-by-state">
-              <label>
-                <div
-                  className={`dropdown ${dropdownStateOpen ? "open" : ""}`}
-                  onClick={toggleDropdownState}
-                >
-                  <div className="dropdown-toggle-state">Sort by State:</div>
-                  <div className="dropdown-menu-state">
-                    {states.map((stateCode) => (
-                      <label key={stateCode} className="dropdown-item">
-                        <input
-                          type="checkbox"
-                          defaultChecked={selectedStateParks.some(
-                            (park) => park.attributes.states === stateCode
-                          )}
-                          onChange={() => handleStateCheckboxChange(stateCode)}
-                        />
-                        {stateCode}
-                      </label>
-                    ))}
-                  </div>
-                </div>
-              </label>
-            </div>
-            <div className="sort-by-designation">
-              <label>
-                <div
-                  className={`dropdown ${dropdownDesignationOpen ? "open" : ""}`}
-                  onClick={toggleDropdownDesignation}
-                >
-                  <div className="dropdown-toggle-des">
-                    Sort by Designation:
-                  </div>
-                  <div className="dropdown-menu-des">
-                    {getUniqueDesignations().map((designation) => (
-                      <label key={designation} className="dropdown-item">
-                        <input
-                          type="checkbox"
-                          defaultChecked={selectedDesignations.includes(
-                            designation
-                          )}
-                          onChange={() =>
-                            handleDesignationCheckboxChange(designation)
-                          }
-                        />
-                        {designation}
-                      </label>
-                    ))}
-                  </div>
-                </div>
-              </label>
+              <StateSort
+                states={states}
+                selectedStateParks={selectedStateParks}
+                handleStateCheckboxChange={handleStateCheckboxChange}
+              />
```

Comment:
Much cleaner. Thank you for doing this!

Reply:
second this, it'll be much easier to parse through this file now!